package/src/components/Message/MessageSimple/MessagePinned.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { Pin } from '../../../icons/Pin';
+
+import {
+  MessageContextValue,
+  useMessageContext,
+} from '../../../contexts/messageContext/MessageContext';
+import { useTheme } from '../../../contexts/themeContext/ThemeContext';
+import { useTranslationContext } from '../../../contexts/translationContext/TranslationContext';
+
+const styles = StyleSheet.create({
+  text: {
+    marginLeft: 5,
+  },
+  view: {
+    display: 'flex',
+    flexDirection: 'row',
+    marginBottom: 8,
+    marginLeft: 30,
+    marginTop: 5,
+  },
+});
+
+import type {
+  DefaultAttachmentType,
+  DefaultChannelType,
+  DefaultCommandType,
+  DefaultEventType,
+  DefaultMessageType,
+  DefaultReactionType,
+  DefaultUserType,
+  UnknownType,
+} from '../../../types/types';
+import { useChatContext } from '../../../contexts/chatContext/ChatContext';
+
+export type MessagePinnedPropsWithContext<
+  At extends UnknownType = DefaultAttachmentType,
+  Ch extends UnknownType = DefaultChannelType,
+  Co extends string = DefaultCommandType,
+  Ev extends UnknownType = DefaultEventType,
+  Me extends UnknownType = DefaultMessageType,
+  Re extends UnknownType = DefaultReactionType,
+  Us extends DefaultUserType = DefaultUserType,
+> = Pick<MessageContextValue<At, Ch, Co, Ev, Me, Re, Us>, 'alignment' | 'message'>;
+
+const MessagePinnedWithContext = <
+  At extends UnknownType = DefaultAttachmentType,
+  Ch extends UnknownType = DefaultChannelType,
+  Co extends string = DefaultCommandType,
+  Ev extends UnknownType = DefaultEventType,
+  Me extends UnknownType = DefaultMessageType,
+  Re extends UnknownType = DefaultReactionType,
+  Us extends DefaultUserType = DefaultUserType,
+>(
+  props: MessagePinnedPropsWithContext<At, Ch, Co, Ev, Me, Re, Us>,
+) => {
+  const { alignment, message } = props;
+  const {
+    theme: {
+      colors: { grey },
+    },
+  } = useTheme();
+  const { t } = useTranslationContext();
+  const { client } = useChatContext();
+  return (
+    <View
+      style={[
+        styles.view,
```

Comment:
`styles.container` is more appropriate. Also these styles should be overridable through theme. You can check the MessageSimple component for example.